client/blocks/reader-excerpt/index.jsx

```diff
@@ -1,6 +1,9 @@
-import { trim } from 'lodash';
+import classNames from 'classnames';
+import { trim, escapeRegExp } from 'lodash';
```

Comment:
I believe we're trying to deprecate lodash in Calypso, so I'd avoid adding new imports, if you can. (p4TIVU-9GZ-p2)

Reply:
the alternative would be escaping it manually with `.replace` like https://stackoverflow.com/questions/3561493/is-there-a-regexp-escape-function-in-javascript. I see there are a few other places in the code we still use `escapeRegExp` from `lodash` so I think that it's probably ok to add this and then when we get around to it, to replace them all with a helper function